tests-web/assets/js/main.js

```diff
@@ -2,45 +2,30 @@ import { makeDropHandler, makeLinkHandler, handleDragOver } from "./files.js";
 import { makeOpenTest, loadTests } from "./tests.js";
 
 const useMap = (map) => {
-    const container = map.getContainer();
-    container.ondrop = makeDropHandler(map);
-    container.ondragover = handleDragOver;
-
-    map.loadLink = makeLinkHandler(map);
-    map.openTest = makeOpenTest(map)
-    console.info("New map created! File drops enabled.", container);
-
-
-    // Here we read the test name from the URL.
-    const q = new URLSearchParams(window.location.search);
-    if (q.has('test')) {
-        const test = q.get('test');
-        console.info("Loading test " + test + " from URL.");
-        map.openTest(test);
-    }
-
-    loadTests();
-}
-
-// Smuggle a reference to the created map, so I can work with it in JS land.
```

Comment:
But I did remove `window.maps` and the indirection for leaflet settings. The interaction before between the Rust leaflet crate and this was kind of confusing; I think the direct thing below is equivalent. Let me know if the map state should live elsewhere, or if we should structure this differently to wire up the "load tests" logic

Reply:
Yes, this is appropriate. I had that indirection because I didn't know the extent of the stuff being done in `yew`.